packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -2836,7 +2836,7 @@ describe('TableView', function () {
       fireEvent.keyDown(document.activeElement, {key: 'ArrowDown'});
       expect(document.activeElement).toBe(within(menu).getAllByRole('menuitem')[1]);
 
-      act(() => table.focus());
+      fireEvent.keyDown(document.activeElement, {key: 'Escape'});
```

Comment:
can you include keyUp as well? i know it's not explicitly needed, but it's closer to real use
I see line 2836 above also doesn't have a keyUp, so technically we've fired two downs with no ups in case we had any global listeners or anything

Reply:
I can, but I'm not going to go back and change every instance in this file (there are a lot).

Reply:
yeah, was worried about that, can address the rest later